src/selectors/app.js

```diff
@@ -422,22 +429,42 @@ const getInvoicesByStatus = (invoices, status) =>
     ? invoices
     : invoices.filter(i => i.status === status);
 
+const getInvoicesByYear = (invoices, year) =>
+  year === getCurrentYear() ? invoices : invoices.filter(i => i.year === year);
```

Comment:
The invoice shape has been changed in the backend, see [here](https://github.com/decred/politeia/blob/master/politeiawww/api/cms/v1/v1.go#L107). 
This has to become:
```suggestion
  year === getCurrentYear() ? invoices : invoices.filter(i => i.input.year === year);
```